config/hardware/addons/accelerometers/adxl345_ebb.cfg

```diff
@@ -1,8 +1,8 @@
 [adxl345]
-cs_pin: EBBCan: ADXL_CS
-spi_software_sclk_pin: EBBCan: ADXL_SCLK
-spi_software_mosi_pin: EBBCan: ADXL_MOSI
-spi_software_miso_pin: EBBCan: ADXL_MISO
+cs_pin: toolhead: ADXL_CS
+spi_software_sclk_pin: toolhead: ADXL_SCLK
+spi_software_mosi_pin: toolhead: ADXL_MOSI
+spi_software_miso_pin: toolhead: ADXL_MISO
```

Comment:
Not sure if it works correctly, but you let a space after the "toolhead: "
Can you remove it to be sure and align this file with all the other files ? Thanks ^^

Reply:
It works, but you're still right : better to keep consistency in the configuration. 
Done. You're welcome . Thanks too ^^